docs/ldml/tr35-keyboards.md

```diff
@@ -1360,37 +1447,52 @@ For example:
 
```

Comment:
I thought we agreed to remove the "after" attribute.

Reply:
true. will remove.